server/core/config/valid/global_cfg.go

```diff
@@ -50,8 +70,57 @@ type StorageBackend struct {
 	S3 *S3
 }
 
+func (s *StorageBackend) GetConfigMap() stow.ConfigMap {
```

Comment:
Why is this struct with the same case statements in every method?

Sounds like this should be an interface with different implementations for each backend.

Reply:
Yup, I have created a `BackendConfigurer` interface which is implemented by all configured backends to return the config to setup stow.